### DISCUSSION TITLE
zh-cn/manual/connection

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#148] [Comments]
# zh-cn/manual/connection

连接设置 ADB 路径 技术细节 自动检测使用的是模拟器的 ADB，但有时自动检测会出现问题，此时就需要手动设置。 强制替换 ADB 是下载谷歌提供的 ADB 后再进行替换，如果自己设置谷歌的 ADB 即可一劳永逸。 使用模拟器提供的 ADB 前往模拟器安装路径，Windows 可在模拟器运行时在任务管理器中右键进程点击 打开文件所在的位置。 顶层或下...

https://docs.maa.plus/zh-cn/manual/connection.html

**Comment by 06jiaozi** [comment]
windows子系统怎么连接

**Reply by JasonHuang79** [reply]
在子系统设置那里有连接地址的，用那个连

**Reply by JasonHuang79** [reply]
在25年3月之后微软就对wsa停止支持了，后续版本的maa也不能正常用wsa了🫠

**Comment by TB-Skadi** [comment]
更了mumu5.0之后老是连接不上，换谷歌adb也没用，连接设置怎么调都不对。最后自己摸索着把mumu的网络桥接关了之后就再没出现连接问题了

**Reply by ABA2396** [reply]
网络桥接默认不开的吧，为什么会打开

**Comment by cornlin1227** [comment]
mac 用的mumuPro模拟器， 之前都用得好好的，更新后突然maa说连接失败了，连接地址都一样但就是连接不上，怎么办

**Reply by Saratoga-Official** [reply]
盲猜你刷理智设置的不支持关卡，mac只要设置不对就会报链接错误，检查链接没用

**Comment by Huan68Meng** [comment]
微软商店的应用宝的ADB连接上了，截图是黑屏状态怎么解，端口也是本地localhost:5555   device端口，emulator-5554   device

**Reply by JasonHuang79** [reply]
你先adb devices看看设备有什么？

**Reply by JasonHuang79** [reply]
因为那个不是maa的官方支持模拟器，所以你可能需要麻烦一点🙂

**Reply by JasonHuang79** [reply]
哦，看了一下，腾讯应用宝是开不了adb的，不要用应用宝用maa哦

**Comment by verioncial** [comment]
蓝叠模拟器 Hyper-V 每次启动端口号都不一样，确实已经按教程勾选 自动检测连接 和 每次重新检测，且修改了gui.json里面的Bluestacks.Config.Keyword和Bluestacks.Config.Path   但是还是连接失败 ，现状是我的D:\Exe\BlueStacks\BlueStacks_nxt_cn\Engine  下面有一个Pie64  文件夹  所以我填的"Bluestacks.Config.Keyword": "bst.instance.Pie64_2.status.adb_port",      bluestacks.conf  我用everything搜索的C:\Users\admin\AppData\Local\Temp\Bst_Logs_hgi2eg2y.5q1\bluestacks.conf  求解答

**Reply by JasonHuang79** [reply]
您最好还是用mumu或者雷电模拟器哦。
用蓝叠的人比较少，所以我们对蓝叠的支持也并不好

**Reply by verioncial** [reply]
感谢回复，我从网上找到了win11清除关闭hyper-V的方法，去掉之后蓝叠的端口号就固定了@玛丽，偷偷揪你头发

**Reply by JasonHuang79** [reply]
@玛丽要@对哦，玛丽是@MistEO 😄

**Comment by StevenGuo1145** [comment]
mumu模拟器打开网络桥接模式，maa也打开网络桥接并且设备编号正确，adb路径无误，连接路径无误，maa无法连接
到模拟器。
后测试过使用adb命令检测adb端口，在有模拟器运行时无设备显示
不开网络桥接方舟莫名其妙下的很慢

**Reply by JasonHuang79** [reply]
你就在下载的时候开不就行了
至于maa和mumu都开网络桥接但连不上的问题，你把这玩意开到外网去了，但是maa检测到是内网的，自然连不上

**Comment by cornlin1227** [comment]
mumu模拟器 mac版有截图增强模式吗，好多作业操作都跟不上

**Comment by LLLTTT303** [comment]
LD增强截图，所有填写都是正确的但是未生效，是9.1.50版本不适用了吗

**Comment by z-zeroo** [comment]
开了截图增强模式后新生息仍然慢了一点。部署时一直差一格就取消了部署，自动战斗正常

**Reply by z-zeroo** [reply]
今天更新后PA-1还是放不下，15可以用

**Comment by 2045700553** [comment]
我的所有连接设置都是正常的，但是开始唤醒就只能打开到mumu模拟器主页面，不能运行对应的安卓设备，这是怎么回事。以下是我的设置信息：

我的连接地址是127.0.0.1:16384（mumu模拟器）
adb路径：D:\Program Files\Netease\MuMu\nx_main\adb.exe
安装路径：D:\Program Files\Netease\MuMu
运行时报错显示：连接失败，尝试重连——重启adbserver，重启模拟器——连接失败，请检查连接设置，尝试重启模拟器和adc，重启电脑

**Reply by 2045700553** [reply]
这个adb端口没有输错，而且我模拟器上只有一个安卓设备
